Resources/Prototypes/materials.yml

```diff
@@ -17,3 +17,12 @@
   electricresistivity: 1.0e+13
   thermalconductivity: 0.9
   specificheat: 840
+
+- type: material
+  id: lasagnamaterial
+  name: Lasagna
+  icon: Objects/Materials/lasagna.png
+  density: 6969
```

Comment:
strongest material known to man: old pasta that was left out on the kitchen counter for 3 days